autogpt_platform/backend/backend/blocks/wordpress/_api.py

```diff
@@ -382,9 +382,6 @@ class CreatePostRequest(BaseModel):
     # Advanced
     metadata: List[Dict[str, Any]] | None = None
 
-    class Config:
-        json_encoders = {datetime: lambda v: v.isoformat()}
```

Comment:
Pydantic knows how to serialize `datetime` objects so this shouldn't be necessary